File: client/iih_stub/iih.ci_stub/iih.ci.mr/nu/obstetrics/birthrec/d/BirthrecInDO.cs

using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.mr.nu.obstetrics.birthrec.d
{
    /// <summary>
    /// BirthrecInDO 的摘要说明。
    /// </summary>
    public class BirthrecInDO : BaseDO {

		public const string TABLE_NAME = "CI_MR_NU_BIRTHRECIN";
		public const string TABLE_ALIAS_NAME = "a0b4";

        public BirthrecInDO() {
        }
		public string Id_birthrecin {
            get { return getAttrVal<string>("Id_birthrecin",null); }
            set { setAttrVal<string>("Id_birthrecin", value); }
        }
		public string Id_birthinfo {
            get { return getAttrVal<string>("Id_birthinfo",null); }
            set { setAttrVal<string>("Id_birthinfo", value); }
        }
		public string Tw {
            get { return getAttrVal<string>("Tw",null); }
            set { setAttrVal<string>("Tw", value); }
        }
		public string Xl {
            get { return getAttrVal<string>("Xl",null); }
            set { setAttrVal<string>("Xl", value); }
        }
		public string Jc {
            get { return getAttrVal<string>("Jc",null); }
            set { setAttrVal<string>("Jc", value); }
        }
		public string Gk {
            get { return getAttrVal<string>("Gk",null); }
            set { setAttrVal<string>("Gk", value); }
        }
		public string Tm {
            get { return getAttrVal<string>("Tm",null); }
            set { setAttrVal<string>("Tm", value); }
        }
		public string Qm {
            get { return getAttrVal<string>("Qm",null); }
            set { setAttrVal<string>("Qm", value); }
        }
		public int? Mb {
            get { return getAttrVal<int?>("Mb",null); }
            set { setAttrVal<int?>("Mb", value); }
        }
		public int? Gscx {
            get { return getAttrVal<int?>("Gscx",null); }
            set { setAttrVal<int?>("Gscx", value); }
        }
		public int? Szy {
            get { return getAttrVal<int?>("Szy",null); }
            set { setAttrVal<int?>("Szy", value); }
        }
		public int? Gsjg {
            get { return getAttrVal<int?>("Gsjg",null); }
            set { setAttrVal<int?>("Gsjg", value); }
        }
		public int? Ssy {
            get { return getAttrVal<int?>("Ssy",null); }
            set { setAttrVal<int?>("Ssy", value); }
        }
		public int? Tx {
            get { return getAttrVal<int?>("Tx",null); }
            set { setAttrVal<int?>("Tx", value); }
        }
		public int? Tedx {
            get { return getAttrVal<int?>("Tedx",null); }
            set { setAttrVal<int?>("Tedx", value); }
        }
		public FDouble Temp {
            get { return getAttrVal<FDouble>("Temp",null); }
            set { setAttrVal<FDouble>("Temp", value); }
        }
		public DateTime? Rq {
            get { return getAttrVal<FDateTime>("Rq",null); }
            set { setAttrVal<FDateTime>("Rq", value); }
        }
		public int? Gsqr {
            get { return getAttrVal<int?>("Gsqr",null); }
            set { setAttrVal<int?>("Gsqr", value); }
        }
        public int Ds {
            get { return getAttrVal<int>("Ds",0);}
            set { setAttrVal<int>("Ds", value); }
        }

        public DateTime? Sv        {
            get { return getAttrVal<FDateTime>("Sv",null); }
            set { setAttrVal<FDateTime>("Sv", value); }
        }
        
        /// <summary>
        /// 返回表名
        /// </summary>
        /// <returns></returns>
        public override string getTableName()
        {
            return "CI_MR_NU_BIRTHRECIN";
        }
        
        /// <summary>
        /// 返回表别名
        /// </summary>
        /// <returns></returns>
        public override string getAliasTableName()
        {
            return "a0b4";
        }
        
        /// <summary>
        /// 返回主键字段名
        /// </summary>
        /// <returns></returns>
        public override string getPKFieldName()
        {
            return "Id_birthrecin";
        }

        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.mr.nu.obstetrics.birthrec.d.BirthrecInDO";
        }

        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[]{"Ds","Sv",  "Id_birthrecin", "Id_birthinfo", "Tw", "Xl", "Jc", "Gk", "Tm", "Qm", "Mb", "Gscx", "Szy", "Gsjg", "Ssy", "Tx", "Tedx", "Temp", "Rq", "Gsqr"};
        }
        
    }
}

File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/mr/nu/obstetrics/birthrec/d/BirthrecInDO.java
package iih.ci.mr.nu.obstetrics.birthrec.d;

import xap.sys.appfw.orm.desc.DescManager;
import xap.mw.coreitf.c.GlobalConst;
import xap.mw.coreitf.i.IDODesc;
import xap.mw.coreitf.d.*;
import xap.mw.core.data.*;
import iih.ci.mr.nu.obstetrics.birthrec.d.desc.BirthrecInDODesc;
import java.math.BigDecimal;

/**
 * 入产房时护理记录 DO数据 
 * 
 */
public class BirthrecInDO extends BaseDO {
	private static final long serialVersionUID = 1L;

	public static final String ID_BIRTHRECIN= "Id_birthrecin";
	public static final String ID_BIRTHINFO= "Id_birthinfo";
	public static final String TW= "Tw";
	public static final String XL= "Xl";
	public static final String JC= "Jc";
	public static final String GK= "Gk";
	public static final String TM= "Tm";
	public static final String QM= "Qm";
	public static final String MB= "Mb";
	public static final String GSCX= "Gscx";
	public static final String SZY= "Szy";
	public static final String GSJG= "Gsjg";
	public static final String SSY= "Ssy";
	public static final String TX= "Tx";
	public static final String TEDX= "Tedx";
	public static final String TEMP= "Temp";
	public static final String RQ= "Rq";
	public static final String GSQR= "Gsqr";
	public static final String DS = "Ds";
	public static final String SV = "Sv";
	
	public String getId_birthrecin() {
		return ((String) getAttrVal("Id_birthrecin"));
	}	
	public void setId_birthrecin(String Id_birthrecin) {
		setAttrVal("Id_birthrecin", Id_birthrecin);
	}
	public String getId_birthinfo() {
		return ((String) getAttrVal("Id_birthinfo"));
	}	
	public void setId_birthinfo(String Id_birthinfo) {
		setAttrVal("Id_birthinfo", Id_birthinfo);
	}
	public String getTw() {
		return ((String) getAttrVal("Tw"));
	}	
	public void setTw(String Tw) {
		setAttrVal("Tw", Tw);
	}
	public String getXl() {
		return ((String) getAttrVal("Xl"));
	}	
	public void setXl(String Xl) {
		setAttrVal("Xl", Xl);
	}
	public String getJc() {
		return ((String) getAttrVal("Jc"));
	}	
	public void setJc(String Jc) {
		setAttrVal("Jc", Jc);
	}
	public String getGk() {
		return ((String) getAttrVal("Gk"));
	}	
	public void setGk(String Gk) {
		setAttrVal("Gk", Gk);
	}
	public String getTm() {
		return ((String) getAttrVal("Tm"));
	}	
	public void setTm(String Tm) {
		setAttrVal("Tm", Tm);
	}
	public String getQm() {
		return ((String) getAttrVal("Qm"));
	}	
	public void setQm(String Qm) {
		setAttrVal("Qm", Qm);
	}
	public Integer getMb() {
		return ((Integer) getAttrVal("Mb"));
	}	
	public void setMb(Integer Mb) {
		setAttrVal("Mb", Mb);
	}
	public Integer getGscx() {
		return ((Integer) getAttrVal("Gscx"));
	}	
	public void setGscx(Integer Gscx) {
		setAttrVal("Gscx", Gscx);
	}
	public Integer getSzy() {
		return ((Integer) getAttrVal("Szy"));
	}	
	public void setSzy(Integer Szy) {
		setAttrVal("Szy", Szy);
	}
	public Integer getGsjg() {
		return ((Integer) getAttrVal("Gsjg"));
	}	
	public void setGsjg(Integer Gsjg) {
		setAttrVal("Gsjg", Gsjg);
	}
	public Integer getSsy() {
		return ((Integer) getAttrVal("Ssy"));
	}	
	public void setSsy(Integer Ssy) {
		setAttrVal("Ssy", Ssy);
	}
	public Integer getTx() {
		return ((Integer) getAttrVal("Tx"));
	}	
	public void setTx(Integer Tx) {
		setAttrVal("Tx", Tx);
	}
	public Integer getTedx() {
		return ((Integer) getAttrVal("Tedx"));
	}	
	public void setTedx(Integer Tedx) {
		setAttrVal("Tedx", Tedx);
	}
	public FDouble getTemp() {
		return ((FDouble) getAttrVal("Temp"));
	}	
	public void setTemp(FDouble Temp) {
		setAttrVal("Temp", Temp);
	}
	public FDateTime getRq() {
		return ((FDateTime) getAttrVal("Rq"));
	}	
	public void setRq(FDateTime Rq) {
		setAttrVal("Rq", Rq);
	}
	public Integer getGsqr() {
		return ((Integer) getAttrVal("Gsqr"));
	}	
	public void setGsqr(Integer Gsqr) {
		setAttrVal("Gsqr", Gsqr);
	}

	public Integer getDs() {
		return ((Integer) getAttrVal(GlobalConst.DELETE_SIGN));
	}
	
	public void setDs(Integer ds) {
		setAttrVal(GlobalConst.DELETE_SIGN, ds);
	}
	
	public FDateTime getSv() {
		return ((FDateTime) getAttrVal(GlobalConst.SYSTEM_VERSION));
	}
	
	public void setSv(FDateTime sv) {
		setAttrVal(GlobalConst.SYSTEM_VERSION, sv);
	}

//	@Override
//	public java.lang.String getParentPKFieldName() {
//		return null;
//	}
  
	@Override
	public String getPKFieldName() {
		return "Id_birthrecin";
	}
	
	@Override
	public String getTableName() {	  
		return "CI_MR_NU_BIRTHRECIN";
	}
	
	@Override
	public IDODesc getDODesc() {
		return DescManager.getInstance().getDODesc(BirthrecInDODesc.class);
	}
	
}